Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/DataSynchronizer.swift

```diff
@@ -17,6 +17,12 @@ extension UpdateResult {
     }
 }
 
+extension DeleteResult {
```

Comment:
[swift5] We will need to remove these to support Swift 5. We should probably use the same strategy we used on Android of creating our own models (SyncDeleteResults, etc.). I'll file a ticket for it.